src/datasources/account/account.datasource.ts

```diff
@@ -59,40 +60,39 @@ export class AccountDataSource implements IAccountDataSource {
       chainId: account.chain_id.toString(),
       emailAddress: new EmailAddress(account.email_address),
       isVerified: account.verified,
-      safeAddress: account.safe_address,
-      signer: account.signer,
+      safeAddress: getAddress(account.safe_address),
+      signer: getAddress(account.signer),
```

Comment:
Theoretically, all addresses returned in this data source should be checksummed. Is there a reason we checksum them again?

Reply:
That's correct. However in order to comply with [the Account  interface](https://github.com/safe-global/safe-client-gateway/blob/775159f7348c78adeb24744dbff8118a93ce8ea2/src/domain/account/entities/account.entity.ts#L11-L12), it does need to be converted to `0x{string}`.

Checksumming the address might prove to be more costly than required indeed however we would still need to validate that it's indeed a `0x{string}` 🤔